roles/python_version/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Get Python version"
+  ansible.builtin.command: python3 --version
+  register: result
+  changed_when: true
+
+- name: "Fail if python version is lower than {{ python_version_expected }}"
+  ansible.builtin.fail:
+    msg: "Python version is {{ result.stdout }}, see https://osism.tech/docs/guides/other-guides/testbed/ for required version."
```

Comment:
I think it makes sense to change this to "Python version is {{ result.stdout }}, expected Python version is {{ python_version_expected }}.". We'll use this role not only in the testbed in the future.